src/Services/Catalog/Catalog.API/Infrastructure/CatalogContextSeed.cs

```diff
@@ -234,7 +234,7 @@ private CatalogItem CreateCatalogItem(string[] column, string[] headers, Diction
                 Description = column[Array.IndexOf(headers, "description")].Trim('"').Trim(),
                 Name = column[Array.IndexOf(headers, "name")].Trim('"').Trim(),
                 Price = price,
-                PictureUri = column[Array.IndexOf(headers, "pictureuri")].Trim('"').Trim(),
+                PictureUri = column[Array.IndexOf(headers, "picturefilename")].Trim('"').Trim(),
```

Comment:
Wouldn't this be better to rename this variable as well ? For e.g 
`PictureFileName = column[Array.IndexOf(headers, "picturefilename")].Trim('"').Trim()`

Reply:
yes,you are right，I rename this variable